src/views/meat-baits-in-traps.njk

```diff
@@ -1,21 +1,57 @@
 {% extends "_base.njk" %}
 
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">
-    Did you use meat bait in your traps?
-  </h1>
+
+  {% if model.meatBaitsError %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [{
+        text: "Select yes if you used meat baits.",
```

Comment:
I would change this error message to `Select yes if you used meat baits in your traps` so it more closely mirrors the question, and there shouldn't be a full-stop at the end.